client/src/apps/Settings/components/SettingsWindow/SettingsWindow.tsx

```diff
@@ -38,7 +38,7 @@ function SettingsWindow(props: SettingsWindowProps) {
     <AppWindow
       title="Settings"
       close={props.close}
-      startSize={{ width: '400px', height: '200px' }}
+      startSize={{ width: '400px', height: '240px' }}
```

Comment:
Just small tweak by the way

Reply:
From what I remember I wanted to also adjust system canvas to behave similarly to https://98.js.org especially when it comes to resizing "desktop" with open app windows. But probably I should report is as a bug or maintenance and should be done in separate PR.